packages/backend/src/vscode-youi-events.ts

```diff
@@ -199,17 +180,61 @@ export class VSCodeYouiEvents implements YouiEvents {
       if (selectedWorkspace === this.messages.open_in_a_new_workspace) {
         void vscode.commands.executeCommand("vscode.openFolder", targetFolderUri);
       } else if (selectedWorkspace === this.messages.add_to_workspace) {
-        const wsFoldersQuantity = _.size(vscode.workspace.workspaceFolders);
+        const wsFoldersQuantity = size(vscode.workspace.workspaceFolders);
         vscode.workspace.updateWorkspaceFolders(wsFoldersQuantity, null, {
           uri: targetFolderUri,
         });
+        if (wsFoldersQuantity === 0) {
+          const workspaceFileUri = this.createNewWorkspaceFile(targetFolderUri.fsPath);
```

Comment:
potential null reference when targetFolderPath==undefined

Reply:
no, this object always returns non empty values